CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.10] - 2025-09-19
+
+### Changed
+- bumped ingress-nginx to v1.13.1 - [link](https://github.com/Azure/aks-app-routing-operator/pull/497)
```

Comment:
nit: call out breaking changes for nginx?

Reply:
done